### DISCUSSION TITLE
Why relationship attributes with lazy="noload" may be missing from inspect().unloaded list?

### DISCUSSION THREAD

**Discussion by tokarenko** [sqlalchemy/sqlalchemy#10854] [Usage Questions]
I have several relationship attributes configured with lazy="noload". When I call inspect() for an instance the attributes are absent from 'unloaded' list. Why may this be the case?

**Comment by zzzeek** [comment]
this depends on the state of the instance and how it got created.   generally "noload" means "do nothing" and it's not really useful for anything these days so you probably don't want to use it.
